test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -660,6 +660,7 @@ def _cleanup(self):
             ftd.extend([f for f in files if bn(f).startswith('pbsscrpt')])
             ftd.extend([f for f in files if bn(f).startswith('pbs.conf.')])
             ftd.extend([f for f in files if p.match(bn(f))])
+            ftd.extend([f for f in du.tmpfilelist])
```

Comment:
I think you can just do 
```suggestion
            ftd.extend(du.tmpfilelist)
```